src/mithril.postgrest.js

```diff
@@ -37,14 +37,16 @@
     var loader = m.prop(defaultState), d = m.deferred();
     loader.load = function(){
       loader(true);
+      m.redraw();
+      m.startComputation();
```

Comment:
My mistake, the end computation will call the redraw at the end, so it is useful even with the background: true